lieer/gmailieer.py

```diff
@@ -707,21 +708,38 @@ def send (self, args):
     self.setup (args, args.dry_run, True, True)
     self.remote.get_labels ()
 
-    if args.message == '-':
-      msg = sys.stdin.buffer.read()
-      fn = 'stdin'
-    else:
-      if os.path.isabs(args.message):
-        fn = args.message
-      else:
-        fn = os.path.join(self.cwd, args.message)
-      msg = open(fn, 'rb').read()
+    msg = sys.stdin.buffer.read()
+    fn = 'stdin'
```

Comment:
We don't need this variable anymore, it will always be stdin.

Reply:
Indeed. dropped `fn` here, and updated the `self.vprint` further down to not print `fn` anymore.